openff/nagl/tests/nn/test_model.py

```diff
@@ -181,7 +179,14 @@ def test_compute_property_networkx(self, am1bcc_model, openff_methane_uncharged)
         assert_allclose(charges, expected, atol=1e-5)
 
     def test_compute_property(self, am1bcc_model, openff_methane_uncharged):
-        charges = am1bcc_model.compute_property(openff_methane_uncharged)
-        charges = charges.detach().numpy().flatten()
+        charges = am1bcc_model.compute_property(openff_methane_uncharged, as_numpy=True)
         expected = np.array([-0.143774, 0.035943, 0.035943, 0.035943, 0.035943])
         assert_allclose(charges, expected, atol=1e-5)
+
+    def test_load(self, openff_methane_uncharged):
+        model = GNNModel.load("openff-gnn-am1bcc-0.0.1-alpha.1.pt")
+        assert isinstance(model, GNNModel)
+
+        charges = model.compute_property(openff_methane_uncharged, as_numpy=True)
+        expected = np.array([-0.111393,  0.027848,  0.027848,  0.027848,  0.027848])
+        assert_allclose(charges, expected, atol=1e-5)
```

Comment:
Based on a meeting with @mattwthompson and @j-wags, we decided not to ship models with NAGL (as is demonstrated in this PR) but to have a separate data repo. There was also some discussion about not using `torch.load` as that just unpickles the model. I was going to look into `load_from_checkpoint` to see if that was any safer, or consider packaging a NAGL-specific format.

Reply:
I switched formats to save a dictionary of model parameters and weights; this at least is more future-proof against updating the classes, although it doesn't solve the pickle problem.